cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/get_compute_fleet_status.erb

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+cat << EOF > get-compute-fleet-status.py
```

Comment:
can we have the `get-compute-fleet-status.py` as a separate file?
It can be deployed under the `/opt/parallelcluster/scripts`folder

Reply:
Yes, changed